IDEAS/Examples/TwinHouses/BuildingO5_Exp1_1Port_PerfectGass.mo

```diff
@@ -0,0 +1,31 @@
+within IDEAS.Examples.TwinHouses;
+model BuildingO5_Exp1_1Port_PerfectGass
+  "Model for simulation of experiment 1 for the O5 building"
```

Comment:
We already have `IDEAS.Examples.TwinHouses.BuildingO5_Exp1` for `Model for simulation of experiment 1 for the O5 building`. Can you adjust the documentation such that it is clear what the purpose of this model is? Please change both this comment and add a note in the documentation section.